extensions/smallrye-openapi/deployment/src/test/java/io/quarkus/smallrye/openapi/test/jaxrs/Auto400Resource.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.smallrye.openapi.test.jaxrs;
+
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+
+@Path("/auto")
+public class Auto400Resource {
+
+    @POST
+    @Path("/400")
+    public void addBar(MyBean myBean) {
+        System.out.println(myBean.bar);
```

Comment:
Can you remove the trace.

Reply:
This code never runs (It's a test case that only test the generated openapi scheme. But I removed it anyway